phpstan-baseline.neon

```diff
@@ -0,0 +1,37 @@
+parameters:
+	ignoreErrors:
+		-
+			message: '#^Creating callable from array\{mixed, non\-falsy\-string\} but it might not be a callable\.$#'
+			identifier: callable.nonCallable
+			count: 2
+			path: src/AuthorizationService.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:can\(\) should return bool but returns Authorization\\Policy\\ResultInterface\|bool\.$#'
+			identifier: return.type
+			count: 1
+			path: src/Controller/Component/AuthorizationComponent.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:canResult\(\) should return Authorization\\Policy\\ResultInterface but returns Authorization\\Policy\\ResultInterface\|bool\.$#'
+			identifier: return.type
+			count: 1
+			path: src/Controller/Component/AuthorizationComponent.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:implementedEvents\(\) should return array\<string, mixed\> but returns array\<int\|string, string\>\.$#'
```

Comment:
Why does it think an array with `int` key is returned?

Reply:
because `$this->getConfig('authorizationEvent')` returns mixed

Reply:
String cast it